core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -314,18 +314,18 @@ public boolean isGameComplete() {
      * @param batch the batch used to draw with
      * @param delta the delta
      */
-    public void draw(SpriteBatch batch, float delta) {
+    public void draw(AbstractGameScreen gameScreen, SpriteBatch batch, float delta) {
         int xoffset = Config.SINGLE_PLAYER_OFFSET;
         if (isSplit) {
             xoffset = Config.SEGMENT_OFFSET * segmentOffset;
         }
         
         //draw the background
-        batch.draw(AssetLoader.bg, xoffset + Config.BORDER_SIZE_SIDES, Config.BORDER_SIZE_BOT,
+        batch.draw(gameScreen.bg, xoffset + Config.BORDER_SIZE_SIDES, Config.BORDER_SIZE_BOT,
```

Comment:
My thought was to pass it down in a hierarchical way and exclude importing AbstractGameClass so we get less coupling. Looking at the Eclipse warning, I guess what I thought of is wrong. Fixed now.